index.js

```diff
@@ -0,0 +1,147 @@
+const day = document.getElementById("js-day"),
+ date = document.getElementById("js-date"),
+ month = document.querySelector(".js-month");
+
+ today = new Date(),
+ MONTH = today.getMonth(),
+ YEAR = today.getFullYear(),
+ PAINTING = "painting";
+
+// 현재 요일, 일자
+function currentDay() {
+    const week = new Array('SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT');
+    day.innerText = week[today.getDay()];
+    date.innerText = today.getDate()
+}
+
+//현재 월, 년
+function currentMonth() {
+    const months = new Array("JAN", "FEB", "MAR", "APR", "MAY", "JUN",
+    "JUL", "AUG", "SEP", "OCT", "NOV", "DEC");
+    const thisMonth = months[today.getMonth()];
+    const thisYear = today.getFullYear();
+    month.innerText = `${thisMonth} ${thisYear}`
+}
+
+// 달력 행&열 추가, 일자 채우기
+function makeCalendar() {
+    let lastDate = new Date(YEAR, MONTH+1, 0).getDate(); // 다음달 0번째 일
+    let firstDay = new Date(YEAR, MONTH, 1).getDay(); //firstDay부터 일자 출력하기 위함
+    let calendar = document.querySelector(".js-calendarBody");
+
+    let row = calendar.insertRow();
+    for(i = 0; i < firstDay; i++){
```

Comment:
`for`문 내에 변수 선언문도 꼭 넣어주세요. 변수 선언문 없이 작성하면 예기치 못한 에러를 발생 시킬 수 있습니다.

```suggestion
    for (let i = 0; i < firstDay; i++) {
```